tests/functional/mhd_shock_with_timevarying_inflow/mhd_shock.py

```
@@ -0,0 +1,270 @@
+    if not diag_path.exists():
+        # diagnostics were not produced (e.g. skipped run) -- nothing to check
+        return
+
+    run = Run(diag_dir)
+
+    checked_any = False
+    for t in timestamps:
+        try:
+            sf = run.GetMHDrho(t)
+        except Exception:
+            continue
```

Ruff (0.15.20)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)